_pages/en/strategy-transform-high-performing-org.md

```diff
@@ -26,29 +26,34 @@ permalink: /strategy-transform-high-performing-org.html
+This **strategy document** is targeted to all stakeholders involved in the day to day operation of IITB.
+This includes both IITB and non-IITB stakeholders such as, but not limited to, Business Functions, Strategic and Service Policy Branch (SSPB), Chief Data Officer (CDO), Integrity Service Branch (ISB), Internal Audit Services Branch, and Shared-Services Canada (SSC).
+The list of stakeholders are listed in the [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
 
-The Guiding Policy, once operationalized, will target all ESDC personnel involved in IT projects, resources and investments.  All teams within IITB are expected to adhere to this policy.
+The **Guiding Policy**, once operationalized, will target stakeholders to always be improving.
+All Employment and Social Development Canada (ESDC) personnel involved in the activities of IITB are expected to adhere to this policy.
 
 ### Business Case
 
-Moving to the digital age requires improving IT’s responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
+Moving to the digital age requires improving IT's responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
 
-To improve IT’s responsiveness, ways must be found to reduce risks associated with its use. This strategy proposes moving ESDC to the point where same day software deployments are possible and a common practice by continuously improving processes inline with the objectives of the branch (IITB) to become a high performing IT organization. The benefits to frequent small deployments, using automated methods, are expected to reduce risk 1, reduce technical debt, increase client satisfaction, and increase overall confidence in the department and its staff.
+The [GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html) and the [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601) as well as the [Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) set a direction for departments to focus on user needs, reuse existing business capabilities, use modern technologies, be agile, work in the open, ..
 
-This strategy capitalizes on existing IT initiatives (such as the IITB Way Forward) to compliment them by adding attention to them, as well as complimenting them with new activities.
+The benefits to the adoption of this Strategy are expected to reduce risk [^1], reduce technical debt over time, increase client satisfaction, and increase overall confidence in the department and its staff to support the needs of ESDC.
```

Comment:
I'm thinking another key benefit would be increased employee satisfaction and feeling of empowerment? Continuous improvement means that each team is taking ownership of improving their own practice and we need to support and enable them in doing so. This includes providing dedicated time to do so but also support trials of new techniques and tools, facilitating the adoption of knowledge but also testing and adoption of trialed technologies.